adaptdl_cli/adaptdl_cli/registry.py

```diff
@@ -49,6 +51,22 @@ def _get_node_ip():
     return ip
 
 
+def fix_etc_hosts():
+    # Correct /etc/hosts entry with the current node IP kubectl is pointing to
+    node_ip = _get_node_ip()
+    host_entry = None
+    if not node_ip:
+        raise SystemExit("Didn't find any nodes.")
+    try:
+        host_entry = socket.gethostbyname(ADAPTDL_REGISTRY_URL.split(':')[0])
+    except socket.gaierror:
+        pass
+
+    if node_ip != host_entry:
+        assert os.system(f"sudo `which hostman` add -f {node_ip} \
```

Comment:
Why `os.system` and not `subprocess.run`?

Reply:
because it's simple. Also I'm not interested in reading out/err etc.

Reply:
In that case, using `subprocess.run` is preferable. It's the way recommended by Python, and nearly as simple. Also it will automatically throw an exception instead of having to use `assert`. The entire statement in the `assert` can be disabled via an environment variable, which we don't have control over.

Reply:
`subprocess.run` takes arguments as list of strings. I'm running small scripts as part of daemon restart which is hard to break down and makes the code/command unreadable. Besides this is completely unrelated to current PR and the problem it fixes.

Reply:
I think you can just pass "shell=True" into `subprocess.run` and it will accept a command as a string. From python docs:
```
>>> subprocess.run("exit 1", shell=True, check=True)
Traceback (most recent call last):
  ...
subprocess.CalledProcessError: Command 'exit 1' returned non-zero exit status 1
```

Reply:
that works. thanks.